### PR TITLE
modify the counting way of ckpts

### PR DESCRIPTION
**Pull request by congee524**
modify the counting way of ckpts

### CODE REVIEW

**Comment by congee524** [comment]
some ckpt files (including some log file) were stored in google drive, so the original method has some problems

**Comment by congee524** [comment]
mmdet_modelzoo_statics

**Comment by ZwwWayne** [comment]
Does it seem that the number of configs is fewer than the number of checkpoints?

**Comment by congee524** [comment]
`fsaf, groie, legacy_1.x` don't have the same number of configs and ckpt, and there are some models share the same config.

But now, there is no need to count config, I will remove later.

**Review by ZwwWayne — APPROVED** [review]

File: docs/stat.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 stats = []
 titles = []
 num_ckpts = 0
+num_configs = 0
```
Comment (ZwwWayne) [inline docs/stat.py:13]: Delete this if this variable is not used.

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**